lib/rules/sort-comp.js

```diff
@@ -87,6 +87,7 @@ module.exports = {
       groups: {
         lifecycle: [
           'displayName',
+          'props',
```

Comment:
I'm a big -1 on including this in the default. Outside of Flow, you would *never* want to declare the `props` instance property anywhere, nor mutate it or assign to it directly.